projects/kubernetes/image-cache/Makefile.pkg

```diff
@@ -1,11 +1,22 @@
-ORG?=linuxkitprojects
-IMAGE=kubernetes-image-cache-$(CACHE)
-NOTRUST=1
-SOURCE=$(BUILDDIR)
-DEPS=$(BUILDDIR)/Dockerfile
-ARCHES=x86_64
+$(BUILDDIR)/build.yml: build.yml.in
+	@sed -e 's/@@CACHE@@/$(CACHE)/g' < $< > $@
```

Comment:
Instead of `sed` would it make sense to provide variables to `linuxkit pkg` via the command line? Could be a improvement done later, though

Reply:
That would involve interpolating all the values and figuring out the semantics of how each one should be dealt with, and handling quoting/escaping. Might be nice but I'm not inclined to do that.